frontend/ui/elements/common_settings_menu_table.lua

```diff
@@ -6,6 +6,7 @@ local Language = require("ui/language")
 local NetworkMgr = require("ui/network/manager")
 local PowerD = Device:getPowerDevice()
 local UIManager = require("ui/uimanager")
+local DocSettings = require("frontend/docsettings")
```

Comment:
Seems the above problem can be fixed with changing this to `require("docsettings")`.

Reply:
You're right, thanks. This makes sense, didn't realize this was how Lua loaded things. Tbh I don't understand why both the above and your correction are accepted in the first place nor why they'd have their own states.

Reply:
Oh, good to know.
So, because the paths given are differents, and even if it's the same file on disk (it is found because of a list of search paths - like $PATH on Unix - and your version is found because it relates to the current working directory), it ends up being 2 different cached objects.